client/packages/common/src/intl/locales/en/common.json

```
@@ -1244,12 +1255,14 @@
   "messages.confirm-delete-assets_other": "This will permanently remove the {{count}} selected assets.",
   "messages.confirm-delete-bundled-item": "This will remove this item from the bundle",
   "messages.confirm-delete-customer-return": "This will permanently delete Customer Return #{{number}}",
-  "messages.confirm-delete-requisitions_one": "This will permanently remove 1 requisition",
-  "messages.confirm-delete-requisitions_other": "This will permanently remove {{count}} requisitions",
   "messages.confirm-delete-document": "This will remove this document.",
   "messages.confirm-delete-generic": "This will permanently remove data",
   "messages.confirm-delete-immunisation-programs_one": "This will permanently remove this Immunization Program.",
   "messages.confirm-delete-immunisation-programs_other": "This will permanently remove the {{count}} selected Immunization Programs.",
+  "messages.confirm-delete-internal-orders_one": "This will permanently remove 1 internal order",
+  "messages.confirm-delete-internal-orders_other": "This will permanently remove {{count}} internal orders",
+  "messages.confirm-delete-internal_orders_few": "This will permanently remove {{count}} internal order",
```

```suggestion
  "messages.confirm-delete-internal_orders_few": "This will permanently remove {{count}} internal orders",
```
Pretty sure that's right for `few`...

---

Oh this came from sorting! But will fix